ui/raidboss/popup-text.js

```diff
@@ -227,8 +244,22 @@ class PopupText {
   }
 
   OnJobChange(e) {
-    this.me = e.detail.name;
-    this.job = e.detail.job;
+    // allow override of player via query parameter
+    // only apply override if player is in party
+    if (Options.PlayerNameOverride !== null &&
+        this.partyTracker.inParty(Options.PlayerNameOverride)) {
```

Comment:
I'm a little bit not sure of this.  Sometimes "being in the party" is quite odd.  Like, a cross world party not in an instance doesn't have anybody in the party.  Or you and your chocobo aren't a party.  Maybe I'm overthinking a first version here, but I wonder a little if we need a "JobOverride" default rather than just not using the override at all if there's not in a party.  This can be punted to a future patch and fixed later if it ever becomes an issue, but just thinking out loud.  I'm fine to merge without changing this unless you have some easy idea that I don't see.

Reply:
Yeah, I think even out-of-dungeon-and-in-separate-overworld-zones cases have issues with not being in the party properly, that's what prompted me to add this in the first place. Maybe this should be moved to its own function and fired here where it was originally as well as in something like zone change?

Reply:
Added `job` parameter in latest changes.